ext/fluxnet_simulations/initial_conditions.jl

```diff
@@ -0,0 +1,205 @@
+"""
+     set_fluxnet_ic!(
+        Y,
+        site_ID,
+        start_date,
+        hour_offset_from_UTC,
+        model::ClimaLand.AbstractLandModel,
+    )
+
+Sets the initial conditions of `Y` using observations from the site `site_ID`, if available,
+using the observations closest to the start_date (in UTC). Since the data from Fluxnet sites
+is provided in local time, we require the offset from UTC in hours `hour_offset_from_UTC`.
+The `model` indicates which how to update it `Y` from these observations,
+via different methods of `set_fluxnet_ic!`.
+"""
+function set_fluxnet_ic!(
+    Y,
+    site_ID,
+    start_date,
+    hour_offset_from_UTC,
+    model::ClimaLand.AbstractLandModel,
+)
+    fluxnet_csv_path = ClimaLand.Artifacts.experiment_fluxnet_data_path(site_ID)
+    (data, columns) = readdlm(fluxnet_csv_path, ','; header = true)
+    # Convert local datetime to time in UTC
+    local_datetime = DateTime.(string.(Int.(data[:, 1])), "yyyymmddHHMM")
+    UTC_datetime = local_datetime .+ Dates.Hour(hour_offset_from_UTC)
+    Δ_date = UTC_datetime .- start_date
+    for component in ClimaLand.land_components(model)
+        set_fluxnet_ic!(Y, data, columns, Δ_date, getproperty(model, component))
+    end
+end
+
+"""
+     set_fluxnet_ic!(Y, data, columns, Δ_date, model::ClimaLand.Soil.EnergyHydrology)
+
+Sets the values of Y.soil in place with:
+- observed value of SWC at the surface at the observation date closest to the start date
+- no ice (θ_i = 0)
+- an internal energy computed using the above θ_l, θ_i, and the temperature of the soil
+  in the first layer, at the observation date closest to the start date. If the soil 
+  temperature is not available, the air temperature is used.
```

Comment:
To visualize which variables are set:
```suggestion
- \vartheta_l: observed value of SWC at the surface at the observation date closest to the start date
- θ_i: no ice (θ_i = 0)
- \rho e_int: an internal energy computed using the above θ_l, θ_i, and the temperature of the soil
  in the first layer, at the observation date closest to the start date. If the soil 
  temperature is not available, the air temperature is used.
```